api/pom.xml

```diff
@@ -44,13 +45,18 @@
         <dependency>
             <groupId>org.jooq</groupId>
             <artifactId>jooq</artifactId>
+            <version>${jooq.version}</version>
         </dependency>
         <dependency>
             <groupId>com.pip.chatbot</groupId>
             <artifactId>database</artifactId>
             <version>0.0.1</version>
             <scope>compile</scope>
```

Comment:
Compile jest domyślne, nie trzeba więc podawać jawnie:) Wersję weźmy z ${project.version}